seed/static/seed/js/services/note_service.js

```diff
@@ -55,15 +55,15 @@ angular.module('BE.seed.service.note', []).factory('note_service', [
     note_factory.create_note = function (org_id, inventory_type, view_id, note_data) {
       var payload = note_data;
       payload.organization_id = org_id;
-      return $http.post('/api/v2.1/' + inventory_type + '/' + view_id + /notes/, payload).then(function (response) {
+      return $http.post('/api/v3/' + inventory_type + '/' + view_id + '/notes/', payload).then(function (response) {
         return response.data;
       });
     };
 
     note_factory.update_note = function (org_id, inventory_type, view_id, note_id, note_data) {
       var payload = note_data;
       payload.organization_id = org_id;
-      return $http.put('/api/v2.1/' + inventory_type + '/' + view_id + /notes/ + note_id + '/', payload).then(function (response) {
+      return $http.put('/api/v3/' + inventory_type + '/' + view_id + '/notes/' + note_id + '/', payload).then(function (response) {
```

Comment:
🤷  ... surprised these worked without the quotes before

Reply:
We got lucky haha
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/toString